AUTHORS.md

```diff
@@ -16,3 +16,4 @@ yourself by creating a pull request (see [CONTRIBUTING.md](CONTRIBUTING.md)).
 - [@chrisgwerder](https://github.com/chrisgwerder)
 - [@0xB767B](https://github.com/0xB767B)
 - Josua Schmid ([@schmijos](https://github.com/schmijos))
+- ([@Tubbles](https://github.com/Tubbles))
```

Comment:
The parentheses around the whole line are probably a copy&paste mistake(?) :see_no_evil: 

```suggestion
- [@Tubbles](https://github.com/Tubbles)
```

Reply:
Haha uuh, yeah.. Thanks for catching that 😆